content/english/pages/donate/_index.md

```diff
@@ -70,16 +63,18 @@ Pawprint Prototyping is a registered California 501(c)(3) non-profit, and operat
 our members and supporters like you.  Donations are fully tax-deductible.  We're grateful for the support of our members, 
 donors, and allies.
 
-Here's three ways you can sustain our mission:
```

Comment:
I think it is important to header this with a specific call to action.  The headers below (donate cash, corporate matching)... are options facilitating that call to action.